cmd/run.go

```diff
@@ -107,6 +110,37 @@ a commandline interface for interacting with it.`,
 			Left:  func() string { return "    init" },
 		}
 
+		// Load plugins
+		cliConf, err := getConfig(cmd.Flags())
+		if err != nil {
+			return err
+		}
+
+		g, _ := errgroup.WithContext(context.Background())
+		if err != nil {
+			return err
+		}
+
+		plugins := []plugin.JavaScriptPlugin{}
+		pluginNames := []string{}
+		for _, pluginPath := range cliConf.Plugins {
+			jsPlugin, pluginErr := lib.LoadJavaScriptPlugin(pluginPath)
+			if pluginErr != nil {
+				return pluginErr
+			}
+
+			// Do the part that actually takes time in a runner group.
+			g.Go(jsPlugin.Setup)
+
+			modules.RegisterPluginModules(jsPlugin.GetModules())
```

Comment:
Hmm we should probably emit an error if there are conflicts in plugin module names?